lib/components/template/layer/ImageLayerEditor.vue

```diff
@@ -93,7 +93,8 @@ v-expansion-panel#image-layer-editor(popout)
       dimensions() { return this.getLayerDimensions(this.layer) },
 
       templateSize() {
-        return this.findTemplateByLayerId(this.layer.id).size
+        let template = this.findTemplateByLayerId(this.layer.id)
+        return template ? template.size : template
```

Comment:
`return template && template.size`

Reply:
and after release when we update everything, i'll enable the "null soaking" operator so we can just do:
`return template?.size`

Reply:
Done!